src/ds/d400/d400-device.cpp

```diff
@@ -187,10 +187,6 @@ namespace librealsense
 
                 synthetic_sensor::open(requests);
 
-                // needed in order to restore the HDR sub-preset when streaming is turned off and on
-                if (_hdr_cfg && _hdr_cfg->is_enabled())
```

Comment:
This piece of code is invoked every time when the streams are opened. Since, FW will throw an error if we try to set it when it is already enabled, removing this code.

Reply:
I am not sure that is what we want, if the user explicitly turned on the hdr , if he closed the stream he expect it to stay on, even if we will block him from changing the exp/gain right?
This will even cause a confusion in the UI IMO, the checkbox will be marked but it will be off in FW.

according to this code Alex answer was wrong and streaming stop turn off HDR preset, right?

Reply:
Quoting Alex's response:
`"SubPreset mechanism stays active in FW (even during start and stop streaming commands) till SETSUBPRESET 0 call."`

So, my understanding from Alex's response is:
- Stream CLOSE doesn't affect HDR.
- So, when the stream is reopened, no need to enable HDR again.

Reply:
It looks like this was not the case when this was implemented originally.
Can we verify this behavior and verify he is right?

Reply:
I verified the following scenario from RS Viewer:
1. HW Reset
2. Set Exp.1 = 32000, Exp.2 = 1
3. Enable HDR
4. Open Depth stream 
5. From frame Metadata, verified that the Actual Exposure values to be 32000 & 1
6. Close and Reopen Depth stream
7. From frame Metadata, verified that the Actual Exposure values are still 32000 & 1

Do you want me to verify any other scenarios?

Reply:
Thanks , that was the required scenario to test